cypress/integration/cucumber/step_definitions/refapp-3.x/04-visit/visit.js

```diff
@@ -0,0 +1,139 @@
+import {Given, After, Before} from 'cypress-cucumber-preprocessor/steps';
+
+let identifier = null;
+let patient = null;
+
+
+Before({tags: '@clinical-visit'}, () => {
+    identifier = cy.generateIdentifier().then((generatedIdentifier)=>{
```

Comment:
Hey @jayasanka-sack , probably s space btn the arrow function would suffice, otherwise well done man

Reply:
Thanks @sherrif10 

fixed with 46a4953